packages/gasket-plugin-command/README.md

```diff
@@ -0,0 +1,201 @@
+# @gasket/plugin-command
+
+Enable plugins to inject new commands to be available for use with the Gasket
+CLI. Executes the `commands` lifecycle in the `configure` hook of the
+`@gasket/plugin-command` plugin. This plugin utilizes [Commander.js] and additional documentation can be found in the documentation.
+
+## Installation
+
+```bash
+npm i @gasket/plugin-command
+```
+
+Update your `gasket.js` plugin configuration:
+
+```diff
+import { makeGasket } from '@gasket/core';
+import pluginCommand from '@gasket/plugin-command';
+
+export default makeGasket({
+  plugins: [
+    // other plugins
++    pluginCommand
+  ]
+})
+```
+
+## Lifecycles
+
+### Commands
+
+Executed in the `configure` hook of `@gasket/plugin-command` if the `gasket` command is present in the `argv`. The `gasket` command is the CLI execution of the `gasket.js` file.
+
+```js
+export default {
+  name: 'test-plugin',
+  hooks: {
+    commands(gasket) {
+      return {
+        id: 'test-plugin-cmd',
+        description: 'Test plugin',
+        action: async () => {
+          console.log('test-plugin');
+        }
+      }
+    }
+  }
+};
+```
+
+### Command Configuration
+
+Commands can be configured with options(flags), arugments, defaults, and parsing functions. The type definitions for the command configuration are also available.
+
+#### Arguments
```

Comment:
heading tweak for consistency ([docs](https://github.com/godaddy/gasket/blob/main/CONTRIBUTING.md#markdown-documentation))
```suggestion
#### Example with Arguments
```